### ISSUE TITLE
[Bug] Recent Colour doesn't display the accurate colour.

### ISSUE THREAD

**Issue by xVanquishx** [JannisX11/blockbench#1459]
Windows 10
Program v4.2.5

If you use the colour picker/generate a palette from a screenshot of Recent Colours, the picked/generated colours are different from what the Recent Colour actually was(Hex Number is vastly different). This doesn't occur if the screenshot is from another program(Only happens to a degree of 1 hex colour R,G, or B).

**Comment by JannisX11** [comment]
Cannot reproduce. Please attach palette and screenshots for reference.

**Comment by xVanquishx** [comment]
Steps to reproduce:
Have a couple colours in Recently Used
![image](https://user-images.githubusercontent.com/29439049/176558518-c87b83f5-f8b4-4283-9898-07d05c3ab356.png)
Go to Picker>Pick Screen Colour>Pick a colour from recently used(In these example the colour next to black is b7e8eb)
![image](https://user-images.githubusercontent.com/29439049/176558802-34cf1d76-022f-4f3b-a907-e9047412b2f3.png)
Tried twice: Once output bde8eb, second time bae8eb(Same Pixel Chosen). 

In Other Attempts the variation in colour was more significant, and this error did not occur when I used an external screenshot/image with the confirmed colour.

As for Generate A Palette: Find a sample online palette: https://www.schemecolor.com/blue-fire.php
Use Colours for painting>Take Screenshot of Recent Colours only containing the Paletted Colours and no bordering content/whitespace.>Paste Screenshot into Texture & Generate A Palette.

Part 2:Using the website's download feature>Download the palette image>Open Image>Screenshot Image cropping out any non-palette colours(Text, Content, Whitespace). I.E:
![bluefirescreenshot](https://user-images.githubusercontent.com/29439049/176559985-7da4a388-2b01-4dc6-a793-f0e1292790da.png)
Copy Paste into Textures>Generate A Palette. (Gave way more accurate colours than from Recent Colours at all merge tolerance levels)
